docker/ui.Dockerfile

```
@@ -1,6 +1,7 @@
 ARG QLEVER_VERSION="0.5.6"
```

Let's also upgrade the CLI here:

```suggestion
ARG QLEVER_VERSION="0.5.7"
```